### PR TITLE
List view: add a modal to delete actions

### PR DESCRIPTION
**Pull request by Pineirin**
* closes: #34

Screenshot from 2022-09-07 10-53-38

### CODE REVIEW

File: invenio_administration/assets/semantic-ui/js/invenio_administration/actions/DeleteModal.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    }
+  };
+
+  render() {
+    const { loading, error } = this.state;
+    const { modalOpen, toggleModal, children, title } = this.props;
+    return (
+      <Modal role="dialog" open={modalOpen}>
+        <Modal.Header as="h2">
+          <Trans defaults="Delete {{title}}" values={{ title: title }} />
```
Comment (alejandromumo) [inline invenio_administration/assets/semantic-ui/js/invenio_administration/actions/DeleteModal.js:47]: Why not use `i18next` directly?

File: invenio_administration/assets/semantic-ui/js/invenio_administration/actions/DeleteModalTrigger.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          apiEndpoint={apiEndpoint}
+          resource={resource}
+          successCallback={successCallback}
+          idKeyPath={idKeyPath}
+          toggleModal={this.toggleModal}
+          modalOpen={modalOpen}
+        >
+          <Modal.Content>
+            <Modal.Description>
+              <Trans
```
Comment (alejandromumo) [inline invenio_administration/assets/semantic-ui/js/invenio_administration/actions/DeleteModalTrigger.js:51]: Same `i18next` question as before.

File: invenio_administration/assets/semantic-ui/js/invenio_administration/actions/DeleteModalTrigger.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                defaults="Are you sure you want to delete {{resourceName}}? "
+                values={{ resourceName: resource[resourceName] }}
```
Comment (alejandromumo) [inline invenio_administration/assets/semantic-ui/js/invenio_administration/actions/DeleteModalTrigger.js:53]: Careful with unprotected accesses `resource[resourceName]` might as well yield `undefined`.

File: invenio_administration/views/base.py [hunk 4]
```diff
@@ -376,6 +377,9 @@ def get(self):
                 "pid_path": self.pid_path,
                 "create_ui_endpoint": self.get_create_view_endpoint(),
                 "list_ui_endpoint": self.get_list_view_endpoint(),
+                "resource_name": self.resource_name
```
Comment (alejandromumo) [inline invenio_administration/views/base.py:380]: In my opinion this creates clutter inside a dictionary's implementation. I don't expect to see a conditional statemente inside a `dict`. 

I suggest you extract this variable before `return self.render`. e.g: 

```python
serialized_schema = self._schema_to_json(schema)
resource_name = self.resource_name or self.id_path

return self.render(...)
```

**Comment by kpsherva** [comment]
additionally, the modals should not be rendered in a loop, which happens now if they are rendered in each row

**Pull request merged**